lib/flutter/flutter_daemon.dart

```diff
@@ -51,13 +63,11 @@ class FlutterDaemonManager implements Disposable {
         .where((String str) => str.startsWith('[') && str.endsWith(']'))
         .map((String str) => str.substring(1, str.length - 1));
 
-      // process.onStderr.listen((String str) {
-      //   _logger.fine('stderr: $str');
-      // });
-
-      // TODO: Do we want to dispose on exit? Notify someone?
       process.onExit.then((_) {
         _daemon.dispose();
```

Comment:
It appears that _initSdk above calls ```_daemon.dispose()``` if ```_daemon != null``` and then reinitializes _daemon. If that happens, will this code be disposing the new instance of _daemon ?